cmd/nodecmd/deploy.go

```diff
@@ -33,6 +34,7 @@ It saves the deploy info both locally and remotely.
 	}
 	cmd.Flags().BoolVar(&subnetOnly, "subnet-only", false, "only create a subnet")
 	cmd.Flags().BoolVar(&avoidChecks, "no-checks", false, "do not check for healthy status or rpc compatibility of nodes against subnet")
+	cmd.Flags().StringSliceVar(&subnetAliases, "subnet-aliases", nil, "additional subnet aliases to be used for RPC calls in addition to subnet blockchain name")
```

Comment:
this was misunderstood: I did not wanted to suggest to add many alias to a blockchain, by to support having many blockchains in the cluster (this is ofc supported) each with its own alias derived from the blockchain name (this is not supported)

Reply:
it will work this way. many blockchains with multiple aliases

Reply:
this is just additional aliases to  subnetName in case user wants `subnetAliases := append([]string{subnetName}, subnetAliases...)`